docs/docs/modeling.md

```diff
@@ -103,7 +103,24 @@ classDiagram
 # Terms and Constraints
 
 ## Owner - オーナー
-
+### Owner
+The Owner is the Owner of ShareHouse
```

Comment:
You might want to mention the `Owner` contains `ShareHouse`.

Reply:
Thank you for your review. I revised that part.